src/mmw/js/src/core/views.js

```diff
@@ -393,6 +396,13 @@ var MapView = Marionette.ItemView.extend({
             map.addControl(new SidebarToggleControl({ model: options.model }));
         }
 
+        if (options.addFitToAoiControl) {
+            map.addControl(new FitToAoiControl({
+                model: options.model,
+                fitToAoi: function() { self.fitToAoi(); }
```

Comment:
[convention] Usually we do this with `_.bind`, something like:

```js
fitToAoi: _.bind(self.fitToAoi, self),
```

Reply:
If I do it this way, then the first argument because a jQuery event, which breaks the argument-less call. Is there a better way to state this that gets around this issue?

Reply:
Ah I see, let's keep it as is then. 👍